code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -80,11 +84,6 @@
 /obj/item/melee/baton/get_cell()
 	return cell
 
-/obj/item/melee/baton/throw_impact(atom/hit_atom)
```

Comment:
Can you add this to the CL as well? Initially from reading the description, I thought it meant that it was a guaranteed stun